src/amo/components/ScreenShots/styles.scss

```diff
@@ -7,10 +7,6 @@ $image-height: $screenshot-height - 20px;
 $screenshot-width: 320px;
 
 .ScreenShots {
-  .pswp-thumbnails {
-    line-height: 0;
-  }
-
   .pswp__caption__center {
```

Comment:
I see you removed the references to the `.pswp-thumbnails` class. I take it that the other classes that start with `.pswp` are still relevant?

Reply:
Yes, `pswp-thumbnails` was part of the old _react_ library, the others seem to be from PhotoSwipe itself. At least, I could find them in the generated HTML.